languages/go/exercises/concept/maps/.docs/instructions.md

```diff
@@ -0,0 +1,46 @@
+A friend of you has an old wholesale store called **Gross Store**, the name came from the quantity of the item that the store sell, it's all in [gross unit][gross-unit]. Your friend asked you to implement a point of sale (POS) system for his store, **but first, you want to build a prototype for it, in your prototype, your system will only record the quantity**. Your friend gave you a list of measurements to help you:
```

Comment:
I really like this story! The prototype addition is a very neat and nifty way to have an "incomplete" spec, excellent idea!

One thing I'm not sure of is the name of the store, as "Gross" also means [extremely unpleasant](https://dictionary.cambridge.org/dictionary/english/gross). Maybe you could come up with a different name?

Reply:
Honesty, I intentionally named it `Gross`, I mean the word is Gross with capital g, not gross, and it also acts as a pun for gross.

Reply:
Ah right, okay. Let's keep it :)